src/fstests/runner.py

```diff
@@ -54,21 +85,17 @@ def run(self):
             self._db.unsubscribe(sub_id)
             return True
 
-class cmd_check(Command):
-    help = 'Check KVM-xfstests environment'
-    args = [Args.db_config]
-    opt_args = [Args.verbose]
-
-    def __call__(self, configs, args):
-        return FstestsRunner(configs, args).check_kvm_xfstest_env()
```

Comment:
Maybe we can actually keep this, so in the future if we want to run it again on a different host or if we're redeploying the VM we can use it as a simple sanity check to confirm the `kvm-xfstests` tool is present.

Reply:
alright!